Data/icons/map_ndb.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What's the license?

Reply:
They are GPL3 https://github.com/Akaflieg-Freiburg/enroute/blob/master/LICENSE

Reply:
That has an important implication - it effectively makes all XCSoar builds GPLv3-only, removing the GPLv2 option.